src/apps/chifra/pkg/utils/utils.go

```diff
@@ -202,12 +202,25 @@ func GetFields(t *reflect.Type, format string, header bool) (fields []string, se
 	return fields, sep, quote
 }
 
-func ToBigInt(str string) big.Int {
+func Str_2_BigInt(str string) big.Int {
 	ret := big.Int{}
+	if str == "0" || str == "0x0" || str == "" {
+		return ret
+	}
 	if len(str) > 2 && str[:2] == "0x" {
 		ret.SetString(str[2:], 16)
 	} else {
 		ret.SetString(str, 10)
 	}
 	return ret
 }
+
+func BigInt_2_Str(bi big.Int) (string, error) {
+	// ret := big.Int{}
+	// if len(str) > 2 && str[:2] == "0x" {
+	// 	ret.SetString(str[2:], 16)
+	// } else {
+	// 	ret.SetString(str, 10)
+	// }
+	return "ret", nil
```

Comment:
This looks like it can generate super weird issues, doesn't it?
It could be just `bi.String()` (or we remove this function entirely)